ui/oopsyraidsy/oopsyraidsy.js

```diff
@@ -458,8 +458,8 @@ class MistakeCollector {
   Translate(obj) {
     if (obj !== Object(obj))
       return obj;
-    if (this.lang in obj)
-      return obj[this.lang];
+    if (this.parserLang in obj)
```

Comment:
This function looks like it is only used for display language.  Can you maybe rename this to TransByDispLang?

Reply:
isn't this `TransByParserLang`?

Reply:
I don't think so.  It's only used for text that is displayed (like reasons for death, etc) and not for regexes.  Am I reading the code incorrectly?

Reply:
ah, yes this is displayLanguage thing. 
ninja rabbit thing was translated by this.
I thought this was translating skill names of boss, but it isn't.